pkg/minishift/network/settings_windows.go

```diff
@@ -41,10 +42,13 @@ var (
 )
 
 func isUsingDefaultSwitch() bool {
-	switchEnvName := "HYPERV_VIRTUAL_SWITCH"
-	switchEnvValue := os.Getenv(switchEnvName)
+	posh := powershell.New()
+	switchNameCommand := fmt.Sprintf("$swname=(Get-VM \"%s\" | Get-VMNetworkAdapter | select SwitchName).SwitchName", profile.GetActiveProfile())
+	_, _, _ = posh.Execute(switchNameCommand)
```

Comment:
Why are we running this i.e. `_, _, _ = posh.Execute(switchNameCommand)` as we are ignoring the output anuway? we are also running it again below `id, _, _ := posh.Execute(switchIdComman`.

Reply:
if using `_, _, _` you do not need the actual ignore options... :-s Just doing `posh.Execute(switchNameCommand)` would be enough

Reply:
Okay.. i did not know that!